### PR TITLE
chore: Add OpenAAVEPosition and CloseAAVEPosition in Operations Regis…

### PR DESCRIPTION
**Pull request by peculiarity**
…try.

### CODE REVIEW

**Review by zerotucks — DISMISSED** [review]

File: scripts/deployment/deploy-create-operations.ts [hunk 1]
```diff
@@ -1,39 +1,59 @@
...
   const system = await utils.getDefaultSystem()
 
-  console.log(`Adding CUSTOM_OPERATION to OperationsRegistry....`)
+  console.log('OperationsRegistry Address:', system.operationsRegistry.address)
+
+  console.log(`Adding ${OPERATION_NAMES.common.CUSTOM_OPERATION} to OperationsRegistry...`)
   await system.operationsRegistry.addOperation(OPERATION_NAMES.common.CUSTOM_OPERATION, [])
+
+  console.log(`Adding ${OPERATION_NAMES.aave.OPEN_POSITION} to OperationsRegistry...`)
+  const openAPositionActions = [
```
Comment (zerotucks) [inline scripts/deployment/deploy-create-operations.ts:26]: The action order for open should be:
* Flashloan
* Set approve
* Deposit in aave
* Borrow aave
* wrap eth
* swap action
* set approval
* deposit aave
* withdraw aave

File: scripts/deployment/deploy-create-operations.ts [hunk 2]
```diff
@@ -1,39 +1,59 @@
...
+  const closeAPositionActions = [
+    getServiceNameHash(CONTRACT_NAMES.common.TAKE_A_FLASHLOAN),
+    getServiceNameHash(CONTRACT_NAMES.common.SET_APPROVAL),
+    getServiceNameHash(CONTRACT_NAMES.aave.DEPOSIT),
+    getServiceNameHash(CONTRACT_NAMES.aave.WITHDRAW),
+    getServiceNameHash(CONTRACT_NAMES.common.SWAP_ACTION),
+    getServiceNameHash(CONTRACT_NAMES.common.SET_APPROVAL),
+    getServiceNameHash(CONTRACT_NAMES.aave.PAYBACK),
+    getServiceNameHash(CONTRACT_NAMES.aave.WITHDRAW),
+    getServiceNameHash(CONTRACT_NAMES.common.UNISWAP_ROUTER),
```
Comment (BackToTheCode) [inline scripts/deployment/deploy-create-operations.ts:49]: This one should be unwrap eth

**Review by zerotucks — CHANGES_REQUESTED** [review]

File: scripts/deployment/deploy-create-operations.ts (hunk 2 above)
Comment (zerotucks) [inline scripts/deployment/deploy-create-operations.ts]: Line 49 should be unwrapETH action

**Review by zerotucks — CHANGES_REQUESTED** [review]

File: scripts/deployment/deploy-create-operations.ts (hunk 2 above)
Comment (peculiarity) [reply scripts/deployment/deploy-create-operations.ts]: Totally a typo.

**Review by zerotucks — APPROVED** [review]

**Pull request merged**